### ISSUE TITLE
IllegalArgumentException in PhpStorm 10.0.3

### ISSUE THREAD

**Issue by nancoder** [idok/scss-lint-plugin#35]
Hi, I'm trying to run scss-lint 1.0.16 in PhpStorm 10.0.3 and a Windows 10 PC, but the event log is throwing the following error:

```
IllegalArgumentException:` Error running ScssLint inspection: Argument for @NotNull parameter 'file' of com/scss/utils/ScssLintRunner.runLint must not be null
SCSS Lint: Error running SCSS Lint inspection: Argument for @NotNull parameter 'file' of com/scss/utils/ScssLintRunner.runLint must not be null`
```

I configured the executable as C:\Ruby22-x64\bin\scss-lint.bat, did I missed something or is a bug?

**Comment by troyt75** [comment]
Hello, I have the same problem but I'm using scss_lint 0.47.1 in PhpStorm 10.0.3 and Windows 7 x64 with the same error.

``Error running ScssLint inspection: Argument for @NotNull parameter 'file' of com/scss/utils/ScssLintRunner.runLint must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/scss/utils/ScssLintRunner.runLint must not be null
	at com.scss.utils.ScssLintRunner.runLint(ScssLintRunner.java)
	at com.scss.ScssLintExternalAnnotator.doAnnotate(ScssLintExternalAnnotator.java:257)
	at com.scss.ScssLintExternalAnnotator.doAnnotate(ScssLintExternalAnnotator.java:51)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.c(ExternalToolPass.java:218)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$500(ExternalToolPass.java:44)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.run(ExternalToolPass.java:160)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:337)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:327)
	at com.intellij.util.ui.update.MergingUpdateQueue$3.run(MergingUpdateQueue.java:271)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:286)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:244)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:234)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:352)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:298)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:295)
	at com.intellij.util.concurrency.QueueProcessor$2$1.run(QueueProcessor.java:110)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.concurrency.QueueProcessor$2.consume(QueueProcessor.java:107)
	at com.intellij.util.concurrency.QueueProcessor$2.consume(QueueProcessor.java:104)
	at com.intellij.util.concurrency.QueueProcessor$3$1.run(QueueProcessor.java:215)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.concurrency.QueueProcessor$3.run(QueueProcessor.java:212)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:366)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:55)
``

**Comment by Triloworld** [comment]
The same error. It's PhpStorm 10.0.3 and the last version of scss_lint (0.47.1) for Windows

**Comment by idok** [comment]
Should be fixed in version 1.0.18
